src/fsharp/FSharp.Core/async.fsi

```diff
@@ -81,7 +94,21 @@ namespace Microsoft.FSharp.Control
         /// If one is not supplied, the default cancellation token is used.</param>
         ///
         /// <category index="0">Starting Async Computations</category>
-        /// 
+        ///
+        /// <example id="start-1">
+        /// <code lang="fsharp">
+        /// printfn "A"
+        ///
+        /// async {
+        ///     printfn "B"
+        ///     do! Async.Sleep(1000)
+        ///     printfn "C"
+        /// } |> Async.Start
+        ///
+        /// printfn "D"
+        /// </code>
+        /// Prints "A", "D" immediately, then "B" quickly, and then "C" in 1 second.
```

Comment:
There's really a race between D and B, with it much more likely that `D` will print first though in theory they might print in either order.  Best to say that

Reply:
I'd considered that, but couldn't think of a real-life scenario in which "B" would print first, as my understanding is that merely calling these Async functions does not change what thread you're on (unless it specifically says so), so the main thread would have to be pre-empted just before the print (should be less than a microsecond window on modern machines, right?) _and_ the asynchronous computation would have to be scheduled and run during that time. Maybe there's another way for this to happen, but I can't think of it right now unless it's something funny with a state machine on the main thread.

In your opinion, is it worth mentioning that "D" is much more likely to print than "B", or is that so system/implementation-specific that it's just noise in the docs?

Reply:
> so the main thread would have to be pre-empted just before the print (should be less than a microsecond window on modern machines, right?) and the asynchronous computation would have to be scheduled and run during that time. 

Yes, this is what would have to happen. It does happen occasionally in practice.

> In your opinion, is it worth mentioning that "D" is much more likely to print than "B", or is that so system/implementation-specific that it's just noise in the docs?

Perhaps "Prints A, then D,B or B,D quickly depending on thread scheduling, then C in 1 second"

That's seems ok